posthog/utils.py

```diff
@@ -354,7 +354,9 @@ def render_template(template_name: str, request: HttpRequest, context: Dict = {}
     context["posthog_app_context"] = json.dumps(posthog_app_context, default=json_uuid_convert)
 
     if posthog_distinct_id:
-        feature_flags = posthoganalytics.get_all_flags(posthog_distinct_id, only_evaluate_locally=True)
+        feature_flags = posthoganalytics.get_all_flags(
+            posthog_distinct_id, only_evaluate_locally=True, groups={"organization": request.user.organization_id}
```

Comment:
Think this might error out for anonymous users (`request.user` can be an anon user), so recommend putting this inside the `if` condition on L345?

(`pk` is an attribute for anon users as well, which is why L344 will not error out)

Reply:
Right, pushed a fix